app/models/text_field_type.rb

```diff
@@ -25,14 +27,24 @@ def mapping_field_name
   end
 
   def text_present
-    errors.add(:text, "must be present") if @text.empty?
+    errors.add(:text, 'must be present') if @text.empty?
   end
 
   def text_length
     validator = LengthValidator.new(validations[:length].merge(attributes: [:text]))
     validator.validate_each(self, :text, text)
   end
 
+  def text_unique
+    unless Field.find_by_name(field_info.name).field_items.jsonb_contains(:data, text: text).empty?
```

Comment:
Yeah I will be pushing up an update that will be: 

```ruby
unless field_info.field_items.jsonb_contains(:data, text: text).empty?
  errors.add(:text, "#{field_info.name} Must be unique")
end
```

since field_info would be the `Field` instance